### ISSUE TITLE
inlineEventCellStyle isn't work in RadCalendar (Angular)

### ISSUE THREAD

**Issue by davidjiangAR** [telerik/nativescript-ui-feedback#563]
### Tell us about the problem
I used the cell styling of sample sdkAngular to testing the events.
Set the inlineEventCellStyle to the RadCalendar but it isn't work.
### Which platform(s) does your issue occur on?
_iOS

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
---HTML----
```
<RadCalendar #calendar id="calendar" locale="en-UK" [eventSource]="events" [eventsViewMode]="eventModel" [monthViewStyle]="monthViewStyle" [weekViewStyle]="weekViewStyle" [monthNamesViewStyle]="monthNamesViewStyle"
                [yearViewStyle]="yearViewStyle" [dayViewStyle]="dayViewStyle" [viewMode]="viewMode"
                [inlineEventCellStyle]="inlineEventCellStyle" (inlineEventSelected)="onInlineEventSelected($event)"></RadCalendar> 
```
----Add the following function in the TS File ---
```
get inlineEventCellStyle():InlineEventCellStyle{
        var cellStyle:InlineEventCellStyle = new InlineEventCellStyle();
        cellStyle.eventTextSize=12;
        cellStyle.eventTextColor = "Red"; 
        //cellStyle.timeTextSize=0;
        
        return cellStyle; 
}
```

**Comment by tsonevn** [comment]
Hi @davidjiangAR,
Thank you for contacting us. 
We reviewed the case and found that there is a bug and the inlineEventCellStyle property cannot be changed via the HTML.

As a workaround, you could use calendar loaded event and to set up the inlineEventCellStyle via code behind as follow. 

HTML
```
<RadCalendar (loaded)="calendarLoaded($event)" [eventSource]="eventSource" [eventsViewMode]="eventsViewMode"  ></RadCalendar> 
```
TypeScript
```
import { Component, OnInit } from "@angular/core";
import {InlineEventCellStyle, CalendarEvent, CalendarEventsViewMode, RadCalendar, CalendarMonthViewStyle} from "nativescript-ui-calendar"
import { Item } from "./item";
import { ItemService } from "./item.service";
import {Color} from "color"
@Component({
    selector: "ns-items",
    moduleId: module.id,
    templateUrl: "./items.component.html",
})
export class ItemsComponent implements OnInit {
    items: Item[];
    public eventSource:Array<CalendarEvent>;
    public eventsViewMode;
    constructor(private itemService: ItemService) { }

    ngOnInit(): void {
        this.items = this.itemService.getItems();
        this.eventSource = this.getCalendarEvents();
        this.eventsViewMode = CalendarEventsViewMode.Inline;
    }
    calendarLoaded(args){
        var calendar = <RadCalendar>args.object;
        console.log("Calendar");
        console.log(calendar.monthViewStyle);
        calendar.monthViewStyle = new CalendarMonthViewStyle();
        (<any>calendar.monthViewStyle).inlineEventCellStyle = this.inlineEventCellStyle();
    }

    inlineEventCellStyle():InlineEventCellStyle{
        var cellStyle:InlineEventCellStyle = new InlineEventCellStyle();
        cellStyle.eventTextSize=12;
        cellStyle.cellBackgroundColor="green"
        cellStyle.eventTextColor = "#0479ca" 

        return cellStyle; 
}
getCalendarEvents(): Array<CalendarEvent> {
    let now = new Date();
    let startDate: Date,
        endDate: Date,
        event: CalendarEvent;
    let colors: Array<Color> = [new Color(200, 188, 26, 214), new Color(220, 255, 109, 130), new Color(255, 55, 45, 255), new Color(199, 17, 227, 10), new Color(255, 255, 54, 3)];
    let events: Array<CalendarEvent> = new Array<CalendarEvent>();
    for (let i = 1; i < 10; i++) {
        startDate = new Date(now.getFullYear(), now.getMonth(), i * 2, 1);
        endDate = new Date(now.getFullYear(), now.getMonth(), (i * 2), 3);
        event = new CalendarEvent("event " + i, startDate, endDate, false, colors[i * 10 % (colors.length - 1)]);
        events.push(event);
        if (i % 3 == 0) {
            event = new CalendarEvent("second " + i, startDate, endDate, true, colors[i * 5 % (colors.length - 1)]);
            events.push(event);
        }
    }
    return events;
}
}
```

**Comment by yassern** [comment]
Works but seems like if I change one style manually, I lost all default style of calendar.

**Comment by NickIliev** [comment]
As a side observation while in NativeScript Core the property is part of `monthViewStyle` in Angular it is the property of RadCalendar (and it is not working via code-behind).

Playground app demonstrating the issue can be found [here](https://play.nativescript.org/)

**Comment by tgpetrov** [comment]
Hi guys,

Here are some updates on this thread.

@davidjiangAR The issue was in the d ts files of the calendar, which were hinting to add an _inlineEventCellStyle_ directly to the calendar. Instead, this style should be added to an instance of _CalendarMonthViewStyle_. I have updated [the playground provided by NickIliev](https://play.nativescript.org/?template=play-ng&id=hNtLe4&v=3) to move the setting of the inline style to the month style and not directly to the calendar. In the nativescript-ui-calendar@4.0.0, the d ts files have been updated so they show the correct way to apply inline event styling is to place an _InlineEventCellStyle_ as a value of the _inlineEventCellStyle_ property of the _CalendarMonthViewStyle_ class (it used to be _RadCalendar_'s property). Since the playground is still using an old version of the plugin, I had to cast the _monthViewStyle_ to _any_ before setting the _inlineEventCellStyle_ but with the latest version it is no longer necessary.

@yassern The issue with the [reset default styles](https://github.com/telerik/nativescript-ui-feedback/issues/576) has been fixed since nativescript-ui-calendar@3.7.0
